qgepqwat2ili/processing_algs/extractlabels_interlis.py

```diff
@@ -0,0 +1,190 @@
+import json
+import os
+import tempfile
+import uuid
+from functools import partial
+
+from PyQt5.QtCore import QCoreApplication
+from qgis import processing
+from qgis.core import (
+    QgsProcessingContext,
+    QgsProcessingFeedback,
+    QgsProcessingParameterBoolean,
+    QgsProcessingParameterEnum,
+    QgsProcessingParameterFileDestination,
+    QgsProcessingParameterVectorLayer,
+    QgsWkbTypes,
+)
+
+from ....processing_provider.qgep_algorithm import QgepAlgorithm
```

Comment:
since this is meant to be used in the plugin, shouldn't we use absolute imports with qgepplugin.qgepqwat2ili ?